worlds/hylics2/__init__.py

```diff
@@ -149,8 +134,8 @@ def fill_slot_data(self) -> Dict[str, Any]:
         slot_data: Dict[str, Any] = {
             "party_shuffle": self.options.party_shuffle.value,
             "medallion_shuffle": self.options.medallion_shuffle.value,
-            "random_start" : self.options.random_start.value,
-            "start_location" : self.start_location,
+            "random_start": int(self.options.start_location != "waynehouse"),
+            "start_location" : self.options.start_location.current_option_name,
```

Comment:
I assume this is meant to use the capitalized forms, with spaces, and the other small changes? So `waynehouse` -> `Waynehouse`, `viewaxs_edifice` -> `Viewax's Edifice`, and so on? I just aske because the docstring says "For display purposes. Worlds should be using current_key."

Reply:
Yes, that's why I used `current_option_name` instead. I did the same thing when implementing an option for A Short Hike a while back.